sdcm/utils/oci_region.py

```
@@ -589,32 +593,50 @@ def create_subnet(
                         ipv6_cidr=subnet_cidr6,
                         public=public,
+                        nic_index=nic_index,
                     )
                 except oci.exceptions.ServiceError as exc:
                     LOGGER.warning("Failed to create a %s subnet: %s", ("public" if public else "private"), exc)
                     time.sleep(2)
                     continue
-            subnet = self.subnet(public=public)
+            subnet = self.subnet(public=public, nic_index=nic_index)
             if subnet is None:
                 raise RuntimeError("Failed to create '{public}' subnet" % public)
```

Ruff (0.16.0)

[error] `%`-format string has 0 placeholder(s) but 1 substitution(s)

(F507)